src/domains/Response/components/ResponseTable/ResponseTable.js

```diff
@@ -80,37 +79,35 @@ function ResponseTable(props) {
   }, [isFormQuiz])
 
   return (
-    <Box display="flex" flex={1} justifyContent="center" overflowX="hidden">
+    <Box overflowX="hidden">
       {loading ? (
         <Spinner width="100%" size="large" />
       ) : (
-        <Row width="100%">
-          <Col>
-            {data?.length > 0 ? (
-              <StyledTable
-                sortOrder
-                columns={columns}
-                dataSource={data}
-                pagination={false}
-              />
-            ) : (
-              <EmptyState
-                description={
-                  emptyStateAnswersDescription ||
-                  'Choose user to display their answers'
-                }
-              />
-            )}
-          </Col>
-        </Row>
+        <Box>
+          {data?.length > 0 ? (
```

Comment:
{data?.length ? () : () }

or Boolean(data?.length)

or !!data?.length

choose one approach and use across all places